### PR TITLE
ci: fix js jobs for running hello world scripts in node

### PR DESCRIPTION
**Pull request by spytheman**
This PR runs again the .js generating/running CI jobs, to prevent regressions.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -104,8 +104,8 @@ jobs:
 #      run: ./v -silent test-compiler
 #    - name: Test v binaries
 #      run: ./v -silent build-vbinaries
-##    - name: Test v->js
-##      run: ./v -o hi.js examples/hello_v_js.v && node hi.js
+    - name: Test v->js
+      run: ./v -b js -o hi examples/hello_v_js.v && node hi.js
```
Comment (spaceface777) [inline .github/workflows/ci.yml:108]: You can just do `./v -b js run examples/hello_v_js.v`.

**Comment by spaceface777** [comment]
Also, this is already tested in `vlib/v/gen/js/jsgen_test.v`, which already compiles and runs everything in `gen/js/tests/`, not sure why we need to test hello world twice.

**Comment by spytheman** [comment]
> vlib/v/gen/js/jsgen_test.v

I did not know about that. 
You are right - it would be better to use that _test.v file, I have not thought about it fully.

The reason I did this PR, was because I discovered that I had introduced a regression in 78e1127 , which went unnoticed till 56ae814 (because main was not being called, node just did nothing, and exited with 0 status code). I tried to do the simplest thing I could and knew how, i.e. uncommented the older lines in the workflow file that used to test the old js backend.

**Comment by spytheman** [comment]
We should probably make vlib/v/gen/js/jsgen_test.v compare at least some of the `node` outputs to known `.out` files.

**Comment by spaceface777** [comment]
Yep, we need more tests for the JS backend, I just added these to have at least something for now. I put some ideas on what to test [here](https://github.com/vlang/v/pull/5147#issuecomment-636507996), will probably implement a separate CI job for this.

**Pull request closed without merging**